lib/index.js

```diff
@@ -178,6 +178,34 @@ function _ensure_cheerio(value) {
     return value;
 }
 
+class DictionaryAsset extends BaseAsset {
+    constructor() {
+        super();
+        this._object_type = "DictionaryObject";
+        this._content_type = "text/html";
+    }
+
+    set_word (value) { this._word = value; }
+    set_word_definition (value) { this._word_definition = value; }
+    set_part_of_speech (value) { this._part_of_speech = value; }
+    set_tags (value) { this._tags = value; }
+
+    to_metadata() {
+        const metadata = super.to_metadata();
+        Object.assign(metadata, {
+            'word': this._word,
+            'wordDefinition': this._word_definition,
+            'partOfSpeech': this._part_of_speech,
+            'tags': [this._tags]
```

Comment:
this line shouldn't be necessary, also `_tags` as a variable implies it should be an array